.github/workflows/build-binaries.yml

```diff
@@ -22,7 +24,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2022, macos-12, ubuntu-20.04]
```

Comment:
macos-13-xlarge is something we pay for, please remove this for build binaries
and remove the change below too, as undeeded